src/EnergyPlus/DXCoils.cc

```diff
@@ -130,10 +130,10 @@ using namespace ScheduleManager;
 
 void SimDXCoil(EnergyPlusData &state,
                std::string_view CompName,                    // name of the fan coil unit
-               HVAC::CompressorOperation const CompressorOp, // compressor operation; 1=on, 0=off
+               HVAC::CompressorOp const compressorOp, // compressor operation; 1=on, 0=off
```

Comment:
Don't use `Operation` when `Op` will do. In general, I am confounded by the choice of which names to contract and which not. Anyways.

Reply:
Why doesn't that logic apply to Compressor? CompOp. I'm trying to think of anything else in HVAC that could be a `Comp`. HVAC:Component, HVAC:Company, nope, it's a compressor.

Reply:
`Comp` is used for `Component` extensively in the code.